src/core_plugins/kibana/public/visualize/editor/sidebar.html

```diff
@@ -6,7 +6,12 @@
     novalidate
   ><!-- see http://goo.gl/9kgz5w -->
 
-    <div css-truncate title="{{indexPattern.id}}" ng-if="vis.type.requiresSearch" class="index-pattern">
+    <div
+      css-truncate
+      aria-label="{{:: 'Index pattern: ' + indexPattern.id}}"
```

Comment:
We should use aria-label here instead of title. Because aria-label overrides any text content, we also need to include the index pattern ID here.